src/Backups/BackupSettings.h

```diff
@@ -32,6 +32,9 @@ struct BackupSettings
     /// Whether the BACKUP command must return immediately without waiting until the backup has completed.
     bool async = false;
 
+    /// Whether the BACKUP will omit similar files (within one backup only).
+    bool deduplicate = true;
```

Comment:
Maybe it's better to invent something more descriptive? Because we already have other types of deduplication in ClickHouse, it can cause confusion. Does `deduplicate_files` sound better, WDYT?

Reply:
>Let's invent a more descriptive name?

Sure.

>Maybe rename it to deduplicate_files, what do you think?

How about `backup_deduplicate` ?

Reply:
Used `backup_deduplicate` for now.

Reply:
It's obvious it's related to backup because it's a backup setting. `deduplicate_files` seems clearer.